pkg/controllers/edgeconnect/consts/consts.go

```diff
@@ -1,24 +1,23 @@
 package consts
 
 const (
-	EdgeConnectUserProvisioned    = "user-provisioned"
-	EdgeConnectContainerName      = "edge-connect"
-	EdgeConnectServiceAccountName = "dynatrace-edgeconnect"
-	EdgeConnectMountPath          = "/etc/edge_connect"
-	EdgeConnectVolumeMountName    = "oauth-secret"
+	EdgeConnectAnnotationSecretHash  = "my-secret-hash"
```

Comment:
I'm open to changing it. Should I add the prefix ` "internal.operator.dynatrace.com/" to it?
I just don't want to import it from dynakube package, better to extract to some common constant.

cc @0sewa0

Reply:
If you use AddAnnotation() from hasher.go the annotation name is in line with all the other occurrences, where we do the same pattern

Reply:
already replied in another thread that we can't use AddAnnotation() here. I've added similar prefix:
https://github.com/dynatrace/dynatrace-operator/blob/4d096200bc7e61cb5c834a6ea1a9328d2affbbbe/pkg/controllers/edgeconnect/consts/consts.go#L4

and 

https://github.com/dynatrace/dynatrace-operator/blob/4d096200bc7e61cb5c834a6ea1a9328d2affbbbe/pkg/controllers/edgeconnect/consts/consts.go#L5